doc/Installation/Installing-LibreNMS.md

```diff
@@ -1,15 +1,17 @@
 source: Installation/Installing-LibreNMS.md
 # Installing LibreNMS
 
-We have some pre-built VirtualBox images you can use to get started:
+##Images
+We have some pre-built VirtualBox images you can use to get started. Please note that these images are pre-built for VirtualBox, hence they might not work with other hypervisors:
```

Comment:
Possibly just adding `hence they might not work with other hypervisors without converting them:`